keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -167,7 +167,8 @@ As a cluster admin, I would like to have cgroup management and log size manageme
 * Handling of zeroed fields
     * It’s possible the configuration of the Kubelet does not handle not specified fields well. Special testing will need to be done for different types to define and ensure conformance of that behavior.
 * Handling of lists
-    * Contention could be found with how lists should be handled (append or overwrite, also see proposal). Consensus should be found and testing performed. 
+    * Contention could be found with how lists should be handled (append or overwrite, also see proposal). Consensus should be found and testing performed.
```

Comment:
when? Is it beta requirement?

Reply:
I don't think we need this anymore. It was there as part of the alpha phase. I will remove that line.